charts/consul/templates/crd-proxydefaults.yaml

```diff
@@ -57,6 +57,33 @@ spec:
           spec:
             description: ProxyDefaultsSpec defines the desired state of ProxyDefaults.
             properties:
+              accessLogs:
+                description: AccessLogs controls all envoy instances' access logging
+                  configuration.
+                properties:
+                  disableListenerLogs:
+                    description: DisableListenerLogs turns off just listener logs
+                      for connections rejected by Envoy because they don't have a
+                      matching listener filter.
+                    type: boolean
+                  enabled:
+                    description: Enabled turns on all access logging
+                    type: boolean
+                  jsonFormat:
```

Comment:
Maybe it's just me, but I was a bit confused on what this field was for until I went down and saw the tests. Initially I thought it could be a bool, not a big issue once you have context but might be nice to have a description that says what it's for (same with textFormat)

Reply:
Good callout. TBH I forgot the comments ended up in the CRD. I'll put a description for both in the next push.